src/components/Button/Button.stories.tsx

```diff
@@ -75,3 +75,30 @@ export const customClass = (): React.ReactElement => (
     Click Me
   </Button>
 )
+
+export const asAnchorTag = (): React.ReactElement => (
+  <Button
```

Comment:
This seems like an example of where using  `as` props are any `string` could be an issue- there's no way to verify, for example, that they are passing types that are valid for the anchor tag.   Or that they even pass a valid html element tag name as a string..